tests/README.md

```diff
@@ -570,6 +570,29 @@ ipdb> test_microvm.help.gdbserver()
 
 You get some instructions on how to run GDB to attach to gdbserver.
 
+## How to run tests with a different version of Firecracker
+
+The integration tests usually compile Firecracker as part of the test
+initialization. But there's an option in case we want to run the tests against a
+different version of Firecracker, for example a previous release:
+
+```sh
+./tools/devtool test -- --binary-dir ../v1.4.1
```

Comment:
What files does one need to put into this binary dir? Just firecracker and the jailer?

Reply:
Yep, clarified.